dashboard/lib/pd/foorm/survey_reporter.rb

```diff
@@ -54,34 +31,97 @@ def self.get_workshop_report(workshop_id)
         parsed_forms,
         questions_to_summarize
       )
-      rollup = Pd::Foorm::RollupCreator.calculate_averaged_rollup(summarized_answers, rollup_question_details)
-
-      result_data[:workshop_rollups] = {
-        questions: rollup_question_details
-      }
+      rollup = Pd::Foorm::RollupCreator.calculate_averaged_rollup(
+        summarized_answers,
+        rollup_question_details,
+        true,
```

Comment:
I went looking for what `true` means, and it made me wonder whether we could used named parameters here to make it more clear?